package.json

```
@@ -208,7 +208,8 @@
     "fsevents": "^2.3.2"
   },
   "resolutions": {
-    "**/ua-parser-js": "<=0.7.28"
+    "**/ua-parser-js": "<=0.7.28",
+    "**/loader-utils": ">=1.4.2"
```

Only 2 loaders are affected so we could maybe try....


```suggestion
    "pegjs-loader/**/loader-utils": ">=1.4.1",
    "po-catalog-loader/**/ua-parser-js": "<=0.7.28"
    
```